composer.json

```diff
@@ -17,10 +17,10 @@
     },
     "require": {
         "php": ">=5.5",
-        "guzzlehttp/guzzle": "^5.3||^6.0.1",
-        "guzzlehttp/psr7": "^1.0",
-        "guzzlehttp/promises": "^1.0",
-        "mtdowling/jmespath.php": "^2.2"
+        "guzzlehttp/guzzle": ">=5.3|>=6.0.1,<7",
```

Comment:
I would do [`~5.3||^6.0.1`](http://semver.mwl.be/#?package=guzzlehttp%2Fguzzle&version=~5.3%7C%7C%5E6.0.1) or [`~5.3||~6.0.1||~6.1`](http://semver.mwl.be/#?package=guzzlehttp%2Fguzzle&version=~5.3%7C%7C~6.0.1%7C%7C~6.1) if you want to avoid the caret sign.